index.js

```diff
@@ -1,23 +1,88 @@
 'use strict';
-const {promisify} = require('util');
 const childProcess = require('child_process');
-const neatCsv = require('neat-csv');
-const sec = require('sec');
+const {promisify} = require('util');
+const pipe = require('multipipe');
+const csv = require('csv');
+const csvHeaders = require('./csv-headers');
+const transform = require('./transform');
 
 const execFile = promisify(childProcess.execFile);
 
-module.exports = async (options = {}) => {
+csv.parse[promisify.custom] = (input, options) => new Promise(resolve => {
+	csv.parse(input, options, (_, records) => resolve(records));
+});
+const parse = promisify(csv.parse);
+
+/**
+ * @typedef Options
+ * @property {Boolean} verbose Make tasklist output more info
+ * @property {Boolean} apps Display store apps
+ * @property {Boolean} services List all service information for each process
+ * @property {String} modules Show tasks that loaded the specified DLL module
+ * @property {String} system The IP address or hostname of the remote machine
+ * @property {String} username The username of the remote machine
+ * @property {String} password The password of the remote machine
+ * @property {Array} filter Filters to pass to the command
+ */
```

Comment:
I don't really want this. It would be better if you could add a TypeScript definition in a follow-up PR, following https://github.com/sindresorhus/typescript-definition-style-guide